### PR TITLE
FTC Dashboard

### PR DESCRIPTION
**Pull request by BD103**
This PR adds [FTC Dashboard](https://acmerobotics.github.io/ftc-dashboard/) support, as well as a sample configuration in the demo API.

### CODE REVIEW

**Comment by BD103** [comment]
This is ready for review now, @cadenceforney.

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/DemoQuadWheels.kt [hunk 1]
```diff
@@ -15,6 +16,17 @@ object DemoQuadWheels : API() {
     private lateinit var bl: DcMotor
     private lateinit var br: DcMotor
 
+    // This is how you define FTC Dashboard configuration.
+    // Uncomment the following line to enable it.
+    // @Config
+    private object DemoQuadWheelsConfig {
+        @JvmField
+        var MAGIC_NUMBER = 5
```
Comment (cadenceforney) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/DemoQuadWheels.kt:24]: I think the magic number should be 4

**Review by cadenceforney — APPROVED** [review]
Looks good to me :)

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/DemoQuadWheels.kt (hunk 1 above)
Comment (BD103) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/DemoQuadWheels.kt:24]: :I

**Pull request merged**